### ISSUE TITLE
Template: Multiple menus in the Focus Order - 2.4.3 Focus Order 

### ISSUE THREAD

**Issue by syarwick** [stephenravenscraft/loyola-template-4.0#89]
Both menus (main and condensed) are in the focus order regardless of which one is visible. this causes "Search" to be in focus twice. 

Ensure content that is not visible does not receive keyboard focus.

**Comment by syarwick23** [comment]
Issue fixed with new template
